### PR TITLE
Version 1 of Contact Us page

### PR DESCRIPTION
**Pull request by PalesaTheCluelessDev**
First webpage - happy for any comments and learnings.

### CODE REVIEW

File: contactUs.php [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+            Swal.fire({
+              title: "Thank you!",
+              text: message,
+              icon: "success"
+            });
+            <?php unset($_SESSION['status']); ?>
+          }
+
+        </script>
+         <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
```
Comment (techgirlshub) [inline contactUs.php]: already added newer version in line 325

File: contactUs.php [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+         <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.6/dist/umd/popper.min.js"></script>
+         <script src="https://stackpath.bootstrapcdn.com/bootstrap/5.3.0/js/bootstrap.min.js"></script>
+
```
Comment (techgirlshub) [inline contactUs.php]: also these 2 imports are duplicates.

File: images/hero.jpg
Comment (techgirlshub) [inline images/hero.jpg:1]: remove unused images

**Review by techgirlshub — CHANGES_REQUESTED** [review]
Please remove PHPMailer-master, there is already an existing phpMailer folder called PHPMailer. used that one. it is inside php folder. alsoplease review the suggestions and 'Resolve conversation' when done.

File: contactUs.php (hunk 2 above)
Comment (PalesaTheCluelessDev) [reply contactUs.php]: fixed locally

File: contactUs.php (hunk 1 above)
Comment (PalesaTheCluelessDev) [reply contactUs.php]: True

File: images/hero.jpg
Comment (PalesaTheCluelessDev) [reply images/hero.jpg:1]: Removed the unused heros

**Comment by PalesaTheCluelessDev** [comment]
> Please remove PHPMailer-master, there is already an existing phpMailer folder called PHPMailer. used that one. it is inside php folder. alsoplease review the suggestions and 'Resolve conversation' when done.

Removed it locally and resolved the other issues.

File: sendContactUsEmail.php [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+            $mail = new PHPMailer(true);
+
+            try {
+                //Server settings
+                //$mail->SMTPDebug = SMTP::DEBUG_SERVER;                            //Enable verbose debug output
+                $mail->isSMTP();                                                   //Send using SMTP
+                $mail->SMTPAuth   = true;                                         //Enable SMTP authentication
+
+                $mail->Host       = 'mail.techgirlshub.co.za';                    //Set the SMTP server to send through
+                $mail->Username   = 'testaccount3@techgirlshub.co.za';            //SMTP username
```
Comment (techgirlshub) [inline sendContactUsEmail.php]: change this and use admin@techgirlshub.co.za, password : Root96void23.(include the fullstop in the password)

File: sendContactUsEmail.php [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+                $mail->SMTPAuth   = true;                                         //Enable SMTP authentication
+
+                $mail->Host       = 'mail.techgirlshub.co.za';                    //Set the SMTP server to send through
+                $mail->Username   = 'testaccount3@techgirlshub.co.za';            //SMTP username        
+                $mail->Password   = 'TestAccount3';                              //SMTP password                            
+                $mail->SMTPSecure = PHPMailer::ENCRYPTION_STARTTLS;             //Enable implicit TLS encryption
+                $mail->Port       = 587;                                        //TCP port to connect to; use 587 if you have set `SMTPSecure = PHPMailer::ENCRYPTION_STARTTLS`
+
+                //Recipients
+                $mail->setFrom('testaccount3@techgirlshub.co.za', 'TechGirlsHub - Test Account 3');
```
Comment (techgirlshub) [inline sendContactUsEmail.php]: Remove Test Account - 3

File: sendContactUsEmail.php [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+                $mail->addAddress('testaccount3@techgirlshub.co.za');
```
Comment (techgirlshub) [inline sendContactUsEmail.php]: change email address to admin email

File: sendContactUsEmail.php [hunk 6]
```diff
@@ -0,0 +1,216 @@
...
+                    .footer a {
+                        color: #007bff;
+                        text-decoration: none;
+                    }
+                </style>
+            </head>
+            <body>
+                <div class="container">
+                    <div class="header">
+                        <h2>Thank You for your Contact Us Request</h2>
```
Comment (techgirlshub) [inline sendContactUsEmail.php]: it should read "Thank you for contacting us"

File: sendContactUsEmail.php [hunk 7]
```diff
@@ -0,0 +1,216 @@
...
+            ';
+
+            // Send the no-reply email to the user
+            $mail->setFrom('no-reply@techgirlshub.co.za', 'TechGirlsHub - No Reply'); // Set no-reply address
+            if ($mail->send()) {
+                $_SESSION['status'] = "Your enquiry has been submitted!";
+                header("Location: {$_SERVER['HTTP_REFERER']}");
+                exit(0);
+            } else {
+                $_SESSION['status'] = "Feedback Response could not be sent to the user. Mailer Error: {$mail->ErrorInfo}";
```
Comment (techgirlshub) [inline sendContactUsEmail.php]: yours should not include feedback

File: sendContactUsEmail.php [hunk 8]
```diff
@@ -0,0 +1,216 @@
...
+            }
+        } else {
+            $_SESSION['status'] = "Feedback could not be sent. Mailer Error: {$mail->ErrorInfo}";
```
Comment (techgirlshub) [inline sendContactUsEmail.php]: feedback remove...

File: sendContactUsEmail.php [hunk 9]
```diff
@@ -0,0 +1,216 @@
...
+        }
+    } catch (Exception $e) {
+        echo "Feedback could not be sent. Mailer Error: {$mail->ErrorInfo}";
```
Comment (techgirlshub) [inline sendContactUsEmail.php]: same

File: sendContactUsEmail.php [hunk 10]
```diff
@@ -0,0 +1,216 @@
...
+                ';
+
+                // Send the email to the admin
+        if ($mail->send()) {
+            // Now send a no-reply email to the user
+            $mail->clearAddresses(); // Clear the previous recipient
+            $mail->addAddress($email, $name); // Add the user's email
+            
+            // Set the subject for the user email
+            $mail->Subject = 'Thank You for Your Feedback';
```
Comment (techgirlshub) [inline sendContactUsEmail.php]: must not read feedback

File: images/underconstruction.jpg
Comment (techgirlshub) [inline images/underconstruction.jpg:1]: do u use this image?

File: images/TGHlogo-noback.png
Comment (techgirlshub) [inline images/TGHlogo-noback.png:1]: remove a logo u are not using, there is a group logo already merged. use that one.

File: contactUs.php [hunk 11]
```diff
@@ -0,0 +1,347 @@
...
+                <!-- Submit Button -->
+                <button type="submit" class="btn btn-primary" name="submitContact" value="Submit" style="padding: 12px; background-color: #333; color: #fff; border: none; border-radius: 4px; cursor: pointer;">Send Message</button>
+            </form>
+        </div>
+    </section>
+
+     <!-- Contact Information Section -->
+     <section style="text-align: center; padding: 20px; background-color: #333; color: #fff;">
+        <h3>Contact Information</h3>
+        <p>Email: testaccount3@techgirlshub.co.za</p>
```
Comment (techgirlshub) [inline contactUs.php]: test account remove

**Review by techgirlshub — CHANGES_REQUESTED** [review]
please review requested changes

File: contactUs.php (hunk 11 above)
Comment (PalesaTheCluelessDev) [reply contactUs.php]: Removed

File: images/TGHlogo-noback.png
Comment (PalesaTheCluelessDev) [reply images/TGHlogo-noback.png:1]: using ---> TechGirlsHub Logo.png

File: images/underconstruction.jpg
Comment (PalesaTheCluelessDev) [reply images/underconstruction.jpg:1]: done

File: sendContactUsEmail.php (hunk 10 above)
Comment (PalesaTheCluelessDev) [reply sendContactUsEmail.php]: reads ---> "Thank You for Your Enquiry"

File: sendContactUsEmail.php (hunk 9 above)
Comment (PalesaTheCluelessDev) [reply sendContactUsEmail.php]: Reads --> "Contact Us Enquiry could not be sent.

File: sendContactUsEmail.php (hunk 8 above)
Comment (PalesaTheCluelessDev) [reply sendContactUsEmail.php]: Reads ---> "Contact Us Enquiry could not be sent."

File: sendContactUsEmail.php (hunk 7 above)
Comment (PalesaTheCluelessDev) [reply sendContactUsEmail.php]: Reads ---> "Contact Us Enquiry could not be sent."

File: sendContactUsEmail.php (hunk 6 above)
Comment (PalesaTheCluelessDev) [reply sendContactUsEmail.php]: Had changed it to "Thank You for contacting TechGirlsHub"....aligned to above

**Review by PalesaTheCluelessDev — COMMENTED** [review]
Done

**Review by MlonziNolwazi — APPROVED** [review]

**Review by techgirlshub — APPROVED** [review]

**Pull request merged**